tox.ini

```diff
@@ -10,6 +10,6 @@ max-line-length = 99
 exclude = tests/__init__.py
 ignore = E722, W503
 
-[testenv:py27-flake8]
-deps = flake8
-commands = flake8 pyuntl setup.py tests
+[testenv:py37-flake8]
+deps=flake8
+commands=flake8 pyuntl setup.py runtests.py tests
```

Comment:
We [got rid of runtests.py](https://github.com/unt-libraries/pyuntl/pull/24/files), so don't need to mention it in the flake8 command.

Reply:
I am fixing this in another branch.